src/pipelinedefinition.cpp

```diff
@@ -1053,17 +1053,17 @@ const tensor_map_t PipelineDefinition::getOutputsInfo() const {
 }
 
 std::shared_ptr<TensorInfo> applyDemultiplexerShapeForTensor(const std::shared_ptr<TensorInfo>& tensorInfo, uint32_t demultiplyCount) {
-    return tensorInfo->createCopyWithEffectiveDimensionPrefix(demultiplyCount);
+    return tensorInfo->createCopyWithEffectiveDimensionPrefix(demultiplyCount ? demultiplyCount : DYNAMIC_DIMENSION);
```

Comment:
This checks for demultiplyCount being 0. I remember talking about 0 and -1 possibility now, is it to be done in separate task? Is there jira for this?

Reply:
Add note to task about DAG to consider accepting -1 as demultiply_count